### PR TITLE
Add details for running cms frontend

### PR DESCRIPTION
**Pull request by neelxie**
# Description
Add instructions for setting up and running the CMS frontend locally 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID
https://trello.com/c/4DlNWGAj

## How Can This Be Tested?
Check out the readme in this branch

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots
Screenshot from 2021-11-29 10-53-35

### CODE REVIEW

**Review by e-ian — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,70 +1,20 @@
...
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+2 - We prefer `Yarn` over NPM but both are supported.
 
-### `yarn test`
+3 - Install the dependencies using 'yarn' or 'npm install' with ```yarn install``` or ```npm install```
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+4 - Create a `.env` file and add the CMS backend URL.
```
Comment (Mubangizi) [inline README.md:17]: Hey @neelxie, I think you might need to specify the keys to be added to the `.env` file something like; `URL_BACKEND`

**Review by Mubangizi — COMMENTED** [review]
@neelxie I don't know if you notice this, but the .env file is on Github, please try to fix this if you can.

File: README.md (hunk 1 above)
Comment (neelxie) [reply README.md]: @Mubangizi this has been fixed sir.

**Review by Mubangizi — APPROVED** [review]
Great work @neelxie

**Pull request merged**